File: RecipeFinalAttempt/app/src/main/java/com/example/recipefinalattempt/SplashPageActivity.java
package com.example.recipefinalattempt;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class SplashPageActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash_page);

        Button welcomeButton = findViewById(R.id.btnWelcome);
        welcomeButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(SplashPageActivity.this, MainActivity.class);
                startActivity(intent);
                finish();
            }
        });
    }
}

File: RecipeApp3/app/src/main/java/com/example/recipeapp/SplashPageActivity.kt
package com.example.recipeapp

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button

class SplashPageActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        findViewById<Button>(R.id.btnWelcome).setOnClickListener {
            var intent = Intent(this@SplashPageActivity, HomePageActivity::class.java)
            startActivity(intent)
            finish()
        }
    }
}